pkg/stdcopy/stdcopy.go

```diff
@@ -54,10 +54,6 @@ func (w *StdWriter) Write(buf []byte) (n int, err error) {
 // `t` indicates the id of the stream to encapsulate.
 // It can be stdcopy.Stdin, stdcopy.Stdout, stdcopy.Stderr.
 func NewStdWriter(w io.Writer, t StdType) *StdWriter {
-	if len(t) != StdWriterPrefixLen {
```

Comment:
Yes you can :). ``t`` has ``StpType``, which is defined by ``type StdType [StdWriterPrefixLen]byte``. I might be completely wrong (and in that case I will revert it) but I removed this check because I don't see how len(t) could be different (because of the definition of ``StdType``).